crates/compute/src/cpu/layer.rs

```diff
@@ -389,25 +386,14 @@ impl<T: TowerFamily> ComputeLayer<T::B128> for CpuLayer<T> {
 	}
 }
 
+unsafe impl<T: TowerFamily> Sync for CpuLayer<T> {}
```

Comment:
Why is this an `unsafe impl`? What's not `Sync` about `CpuLayer` by default?

Reply:
This is because of `PhantomData<T>` requires T to be `Sync`